### DISCUSSION TITLE
Update 3.6.5 -> 3.7.0 from WebUI: wrong download link

### DISCUSSION THREAD

**Discussion by wrohdewald** [emsesp/EMS-ESP32#2253] [General]
### DESCRIPTION

I  have 3.6.5 installed. When checking for updates, the download link for 3.7.0 points to

https://github.com/emsesp/EMS-ESP32/releases/download/v3.7.0/EMS-ESP-3_7_0-ESP32.bin

but that does not exist.

I can see 

https://github.com/emsesp/EMS-ESP32/releases/download/v3.7.0/EMS-ESP-3_7_0-ESP32-16MB+.bin
https://github.com/emsesp/EMS-ESP32/releases/download/v3.7.0/EMS-ESP-3_7_0-ESP32-16MB.bin
https://github.com/emsesp/EMS-ESP32/releases/download/v3.7.0/EMS-ESP-3_7_0-ESP32-4MB.bin

### REQUESTED INFORMATION

How can I find out which one I need?

The dev download for 3.7.1-dev.12 points to an existing file.

**Comment by MichaelDvP** [comment]
depends on the hardware you use, see https://download.emsesp.org/

**Comment by proddy** [comment]
@wrohdewald this is because in 3.6.5 the download URL generated in the web code doesn't know which board you are using, if on a BBQKees gateway (4MB, 16MB, PSRAM or not). There's nothing we can do about this unfortunately. Which is why we created the download.emsesp.org page to help people out.
